playbooks/files/rax-maas/plugins/swift-recon.py

```diff
@@ -40,34 +40,50 @@ class CommandNotRecognized(maas_common.MaaSException):
     pass
 
 
+def getcontainer(pattern, containers):
```

Comment:
I wasn't sure about this as container=None initially.  Also wasn't sure if you were guaranteed to find the search pattern in the list of containers.  I was simply preserving the context of the original code.  Is the state/context still None after iteration through containers?  I figured safe rather than sorry?

Reply:
It's functionally the same thing, with the existing code, if there are no matches this function still returns `None`.